Dockerfiles/agent/windows/amd64/Dockerfile

```diff
@@ -1,4 +1,9 @@
 ARG BASE_IMAGE=mcr.microsoft.com/powershell:lts-windowsservercore-1809
+FROM ${BASE_IMAGE} as unzipper
+SHELL ["pwsh", "-Command", "$ErrorActionPreference = 'Stop';"]
+COPY datadog-agent-latest.amd64.zip uncompress-zip.ps1 ./
+RUN . ./uncompress-zip.ps1

```

Comment:
There's no need to run this on a separate image. We could run it on the main one and uninstall 7zip after we are done to not leave things behind.

Reply:
I think multi-stage makes the Dockerfile a bit less comprehensible, but there's nothing wrong other than that.